extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -159,6 +159,7 @@ public function testRenderInfoboxDataProvider() {
 						'width' => '400',
 						'height' => '200',
 						'thumbnail' => 'http://thumbnail.jpg',
+						'thumbnail2x' => 'http://thumbnail2x.jpg',
 						'media-type' => 'image',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "media-type" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)